### PR TITLE
RAINCATCH-1257 introduction to intergrations

### PR DESCRIPTION
**Pull request by austincunningham**
## Motivation:
Introduction to integrations like sync keycloak (just mentiion)

## What:
We will need to build top level concept that explains what kind of integrations we have and why we need them. We need to explain:
-  What projects raincatcher integrates
-  What value integration give (introduction)

- [ ] Documentation

### CODE REVIEW

File: docs/workforce-management-framework/topics/con-wfm-intergrations.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Authentication and Authorisation are the primary focus of these xref::{context}-con-introducing-securityfeatures[security intergrations].
+
+=== Keycloak
+
+Keycloak is a full identity management solution which runs on a seperate server. 
+For more information see xref::{context}-ref-keycloak-securitystrategy[Keycloak intergration].
+
+=== Passport.js
+
+Passport.js is a light weight secrutiy solution that runs by default when {Feature} is installed.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/con-wfm-intergrations.adoc:19]: secrutiy

File: docs/workforce-management-framework/topics/con-wfm-intergrations.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
+= {Feature} intergration
+
+An intergration is the use of existing opensource projects to solve specific needs for {Feature}. 
+{Feature} has a number of intergrations that leverage other opensource projects for there expertises in paticular fields.
```
Comment (witmicko) [inline docs/workforce-management-framework/topics/con-wfm-intergrations.adoc:4]: >for there expertises in paticular fields

for their expertise in particular fields

File: docs/workforce-management-framework/topics/con-wfm-intergrations.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
+= {Feature} intergration
+
+An intergration is the use of existing opensource projects to solve specific needs for {Feature}.
```
Comment (witmicko) [inline docs/workforce-management-framework/topics/con-wfm-intergrations.adoc:3]: >existing opensource projects

Im guessing that we are not limited to opensource only?

**Comment by wtrocki** [comment]
@austincunningham Can you rebase and squash your commits there to provide good baseline for @jstaffor to work with. Ideally we looking for single commit.

File: docs/workforce-management-framework/topics/con-wfm-intergrations.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
+= {Feature} intergration
+
+An intergration is the use of existing opensource projects to solve specific needs for {Feature}. 
+{Feature} has a number of intergrations that leverage other opensource projects for there expertises in paticular fields.
+This enabled the inclusion of domain expertises in the areas of security and sync in the developement of {Feature}.
```
Comment (witmicko) [inline docs/workforce-management-framework/topics/con-wfm-intergrations.adoc:5]: >inclusion of domain expertises in 

domain expertise

**Review by witmicko — COMMENTED** [review]
good in general, couple comments

;) https://github.com/Microsoft/vscode-spell-check

**Comment by wtrocki** [comment]
@austincunningham Dropping tech-approved as rebase required.

File: docs/workforce-management-framework/topics/con-wfm-intergrations.adoc (hunk 3 above)
Comment (austincunningham) [reply docs/workforce-management-framework/topics/con-wfm-intergrations.adoc:3]: At the moment all open source and ideally to continue that way.

**Comment by wtrocki** [comment]
PR created on upstream. 
@austincunningham let's follow up the same pattern next time

**Pull request closed without merging**